cmd/pw_router/worker.go

```diff
@@ -194,6 +199,13 @@ func (w *worker) handleMsg(msg []byte) error {
 		// we have existing data, check to make sure we
 		record, _ := w.rabbit.syncSamples.Load(update.Icao)
 		tRecord := record.(planeLocationLast)
+
+		//check if this is a removed record and purge it from the cache
+		if update.Removed {
```

Comment:
If I understand this correctly, we swallow this message here

I would argue that this is an important message to pass onto the front end so that it can do its own garbage collection